docs/release-notes/release-notes-0.18.0.md

```diff
@@ -330,6 +332,10 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
 * The [`estimateroutefee`](https://github.com/lightningnetwork/lnd/pull/8136)
   subcommand now gives access to graph based and payment probe fee estimation.
 
+* [`sendcoins` now takes in an optional utxo flag](
+ https://github.com/lightningnetwork/lnd/pull/8516). This allows users to
+specify the coins that they want to use as inputs for the transaction.
```

Comment:
nit: should indent with two spaces to make it visually belong to the same list item/paragraph.